core/src/test/scala/unit/kafka/server/BrokerFeaturesTest.scala

```diff
@@ -95,10 +97,20 @@ class BrokerFeaturesTest {
 
     val expectedFeatures = Map[String, Short](
       MetadataVersion.FEATURE_NAME -> MetadataVersion.latestTesting().featureLevel(),
+      ServerFeatures.TRANSACTION_VERSION.featureName() -> ServerFeatures.TRANSACTION_VERSION.latestTesting(),
       "kraft.version" -> 0,
       "test_feature_1" -> 4,
       "test_feature_2" -> 3,
       "test_feature_3" -> 7)
     assertEquals(expectedFeatures, brokerFeatures.defaultFinalizedFeatures)
   }
+
+  @ParameterizedTest
+  @ValueSource(booleans = Array(true, false))
+  def ensureDefaultSupportedFeaturesRangeNotZeroZero(unstableVersionsEnabled: Boolean): Unit = {
+    val brokerFeatures = BrokerFeatures.createDefault(unstableVersionsEnabled)
+    brokerFeatures.supportedFeatures.features().values().forEach { supportedVersionRange =>
+      assertFalse(supportedVersionRange.min() == 0 && supportedVersionRange.max() == 0)
```

Comment:
Perhaps it's better to only test `supportedVersionRange.max() == 0` since we want to disallow ranges like (1,0) too? If so, we want to adjust the method name accordingly. Ditto for a similar test in [QuorumFeaturesTest.java](https://github.com/apache/kafka/pull/16183/commits/dc34d935673d6c4eb9373dfea092795ce99a1d8e#diff-7160ed0388ecc53d2af1e7b722f59a25f95475cd36baf4a14e0c038a5473ab97).

Reply:
Sure. I can do that.